coda-src/venus/vproc_vfscalls.cc

```diff
@@ -1473,7 +1473,7 @@ void vproc::read(struct venus_cnode * node, uint64_t pos, int64_t count)
         return;
     }
 
-    if (pos >= f->Size()) {
+    if (pos > f->Size()) {
```

Comment:
Haven't thought it through, but I think  pos >= Size() is correct (because position/index is 0-indexed)

Reply:
Right! Rolling this back.